gTea/app/src/main/java/com/nechaieva/gtea/utils/LevenshteinOrderProcessor.java

```diff
@@ -11,7 +11,13 @@
 public class LevenshteinOrderProcessor implements OrderProcessor {
 
     private List<String> menu;
-    public final int DEFAULT_SENSITIVITY = 3; // The threshold for string distances.
+
+    /**
+     * The threshold for string distances.
+     * Represents the number of typos the user can make before the query will be rejected.
```

Comment:
"number of typos" seems rather imprecise given that there is. a more precise definition available